packages/commonwealth/server/routes/status.ts

```diff
@@ -118,6 +119,7 @@ export const getUserStatus = async (models: DB, user: UserInstance) => {
       isAdmin,
       disableRichText,
       communities: userCommunities?.[0] || [],
+      starredCommunities: userCommunities?.[0] || [],
```

Comment:
Same issue as the other PR. `starredCommunities` is deprecated and should be removed from the type.